<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view::PlayerCardWidget</class>
 <widget class="QWidget" name="view::PlayerCardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>211</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">:hover {
	background-color: #808080;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="wgtColor" native="true">
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lblName">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Player Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMoney">
        <property name="text">
         <string>$ Money</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTrains">
        <property name="text">
         <string>Railways:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgtJail" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblJailTurns">
           <property name="text">
            <string>X turns in jail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="imgJail">
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../assets/images/jail-icon.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
